spanish-translations.md

```diff
@@ -0,0 +1,19 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our
+tutorials (script and subtitles). How we made this possible? You may think that
```

Comment:
Add another sentence with a link to a Spanish page to see an example. And probably we should mention *how* to switch to Spanish - perhaps that could be the image of the post (showing... maybe a video player with the captions drop down expanded or some other "Spanish" link from somewhere on the site)?

Also, I think you're under-selling it! This includes the ability to click any spanish sentence in the script and jump to that part of the video, just like English.

Reply:
I didn't add a section showing how to change language, I think that's intuitive but let me know if you think otherwise